### ISSUE TITLE
Webcams dissappear during session, while session continues correctly. Not possible to activate webcams again.

### ISSUE THREAD

**Issue by matobln** [bigbluebutton/bigbluebutton#19423]
**Describe the bug**
Since Upgrade to 2.7.1 we experience the problem that by users activated webcams dissappear during a bigbluebutton session, while the session with audio and deskshare continues to work correctly. webcams dissappear in all currently running vrooms.

It is not possible to reactivate the webcams in the currently running sessions.
We have to reboot the server to solve the issue, until it happens again.
Times differ between twice a day this can happen to 2 weeks without problems.

We currently run 2.7.3 and the problem still continues.
Small BigBlueButton Server with not more than 2-3 parallel vrooms with not more that 50-60 concurrent users in total.
Workload of the server is low.

I checked with the server provider. He says he is not throttling nor doing anything to the 1 Gbit Connection to server.

Any ideas what could cause this?

Thanks

**Comment by hostbbb** [comment]
can you look in bigbluebutton.properties.

Set recordFullDurationMedia=true ,  it defaults to false.    this fixed a similar issue that was in 2.7.1 ,  but i thought was fixed.

See if the problem still occurs once you have it set to true..     And let us know.

**Comment by tibroc** [comment]
Yes, this seems to be the behavior mentioned in https://github.com/bigbluebutton/bigbluebutton/issues/19085

**Comment by Ithanil** [comment]
It has been fixed since two months now, but unfortunately no new release has been made since then. But it looks like 2.7.4 is _finally_ just around the corner.
